apps/web/src/components/ColorSchemeModeToggle/ColorSchemeModeToggle.tsx

```diff
@@ -0,0 +1,14 @@
+import { Flex, Switch, useColorMode } from "@chakra-ui/react";
+
+export const ColorSchemeModeToggle = () => {
+  const { colorMode, toggleColorMode } = useColorMode();
+
+  const switchText = colorMode === "dark" ? "Light mode" : "Dark mode";
+
+  return (
+    <Flex alignItems="center" gap="14px">
+      {switchText}
```

Comment:
shall it be maybe wrapped in a `Text`?

Reply:
AFAIK <Text/> in chakra ui stands for p tag, so this is not the case